packages/fakta-beregning-redesign/src/components/avklareAktiviteter/AvklareAktiviteterField.tsx

```diff
@@ -10,6 +10,7 @@ import {
   BeregningsgrunnlagTilBekreftelse,
   Vilkarperiode,
 } from '@navikt/ft-types';
+import { SubmitButton } from '@navikt/ft-frontend-internal-components';
```

Comment:
dette vil nok føre til 404-feilen igjen

Reply:
@vebnor jeg tror ikke det. Grunnen er denne endringen

```diff
    rollupOptions: {
-      external: Object.keys(dependencies),
+      external: Object.keys(peerDependencies),
    },
```
Grunnen til at det ikke fungerte tidligere var fordi `rollupOptions.external` ekskluderte alle avhengighetene 
 under `dependencies` i package.json. 

Og siden `@navikt/ft-frontend-internal-components` lå i dependencies ble den eksludert.  Hvis man bygger pakkene nå finner man definisjonen av SubmitButton i `dist/`

Reply:
men egentlig tenker jeg at SubmitButton kan flyttes til `form-hooks/` da vi har andre form-relaterte komponenter der

Reply:
ahh, da så 👍 da kan vi godt prøve det. Det er ikke så nøye for meg hvor den knappen ligger, jeg bare ville ikke ha den i form-hooks siden den antageligvis ikke kommer til å brukes noe annet sted enn intern her, men så ble det veldig mye mer styrete enn forventet 😄